CocktailRecipes/CocktailRecipes/Features/Home/HomeViewController.swift

```diff
@@ -176,6 +188,10 @@ extension HomeViewController: SectionHeaderDelegate {
         let sectionType = presenter.sectionType(for: sectionIndex)
         let dataSource = presenter.dataSource(for: sectionType)
 
-        #warning("push to specific section here")
+        let drinkChartPresenter = DrinkListPresenter(drinks: dataSource)
```

Comment:
rename the properties and the one below it too

Reply:
Done.